format/index.md

```diff
@@ -1287,6 +1309,8 @@ primary data, the same request URL can be used for updates.
 
 The `PATCH` request **MUST** include a single resource object as primary data.
 The resource object **MUST** contain `type` and `id` members.
+The resource object **MUST** contain `revision` member if the server supplied
+the `revision` member in a `GET` request.
```

Comment:
We can't make a **MUST** level requirement of clients based on the server implementation. If a 1.1 server includes `revision`, then a 1.0 client should still work without including `revision`.